### PR TITLE
implemented profile/settings page

### PR DESCRIPTION
**Pull request by mkenlo**
In this PR, I added a new screen to display User information and its preferences

### CODE REVIEW

File: lib/config.dart [hunk 1]
```diff
@@ -4,6 +4,8 @@ const String recordStorage = "/sdcard/Lingua";
 
 const String defaultTargetLang = "Yemba";
 
+const String defaultSourceLang = "francais";
```
Comment (evindj) [inline lib/config.dart:7]: capital F in "francais"

File: lib/screens/profile_screen.dart [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+        }
+      },
+    );
+
+    showDialog<void>(
+        context: context,
+        barrierDismissible: true,
+        builder: (BuildContext context) {
+          return AlertDialog(
+              title: Text("Change Language"), content: languages);
```
Comment (evindj) [inline lib/screens/profile_screen.dart:104]: not a big deal but probably want to localize this string.

**Review by evindj — APPROVED** [review]
Thumbs up getting better fast!!!

File: lib/screens/profile_screen.dart (hunk 2 above)
Comment (mkenlo) [reply lib/screens/profile_screen.dart:104]: Will do that

File: lib/config.dart (hunk 1 above)
Comment (mkenlo) [reply lib/config.dart:7]: for uniformity, I decided to use languages in small case letters.

Comment (mkenlo) [reply lib/config.dart:7]: `But I will display by capitalizing the word.`

**Pull request merged**